### PR TITLE
Features reprojection

### PR DESCRIPTION
**Pull request by procrastinatio**
This PR 
* SQLAlchemy and Geoalchemy update (security, https://github.com/geoadmin/mf-chsdi3/network/alert/requirements.txt/SQLAlchemy/open)
* All feature/geom transformations outside the DB (hopefully fixing #3172 and  #2965)

[Demo](https://mf-chsdi3.int.bgdi.ch/sqlalchemy_update)

Identify in **WGS1984**...
https://mf-chsdi3.int.bgdi.ch/sqlalchemy_update/rest/services/api/MapServer/ch.bafu.nabelstationen/RIG,LAU?sr=21781&geometryFormat=geojson
...and **LV03**
https://mf-chsdi3.int.bgdi.ch/sqlalchemy_update/rest/services/api/MapServer/ch.bafu.nabelstationen/RIG,LAU?sr=4326&geometryFormat=geojson

### CODE REVIEW

File: chsdi/lib/helpers.py [hunk 1]
```diff
@@ -292,25 +303,91 @@ def imagesize_from_metafile(tileUrlBasePath, bvnummer):
...
+    else:
+        return _transform_shape(geom, srid_from, srid_to, rounding=rounding)
+
+
+def _transform_coordinates(coordinates, srid_from, srid_to, rounding=True):
+    if len(coordinates) % 2 != 0:
+        raise ValueError
+    new_coords = []
+    coords_iter = iter(coordinates)
+    for pnt in izip(coords_iter, coords_iter):
```
Comment (boecklic) [inline chsdi/lib/helpers.py]: maybe add a comment about format of `coordinates` (flat array, list of tuples,..?), it's not immediately obvious, how the `iter`/`izip` trick works

File: tests/integration/test_features_service.py [hunk 2]
```diff
@@ -339,6 +341,26 @@ def test_feature_with_callback(self):
...
+    def test_feature_esirjson_geom_supported_srs(self):
+        bodId = 'ch.bafu.bundesinventare-bln'
+        featureId = self.getRandomFeatureId(bodId)
+
+        for srs in SUPPORTED_OUTPUT_SRS:
+            resp = self.testapp.get('/rest/services/ech/MapServer/%s/%s' % (bodId, featureId), params={'sr': srs, 'geometryFormat': 'esrijson', 'returnGeometry': 'true'}, status=200)
+            self.assertEqual(resp.content_type, 'application/json')
+            self.assertEqual(resp.json['feature']['id'], featureId)
+            self.assertEsrijsonFeature(resp.json['feature'], srs)
+
```
Comment (boecklic) [inline tests/integration/test_features_service.py]: maybe add a test for no srs specified (to check correct default value)?

File: chsdi/models/__init__.py [hunk 3]
```diff
@@ -65,7 +65,8 @@ def set_models_srid(models, srid):
         return models
     ms = []
     for m in models:
-        m.set_geometry_srid_out(srid)
+        # TODO remove completely
+        # m.set_geometry_srid_out(srid)
```
Comment (boecklic) [inline chsdi/models/__init__.py]: maybe the class method should be removed as well (or at least commented)?

File: requirements.txt [hunk 4]
```diff
@@ -49,7 +49,7 @@ simplejson==3.16.0
 six==1.11.0
 Sphinx==1.2.3
 sphinx-rtd-theme===0.1.6-ga3
-SQLAlchemy==1.0.14
+SQLAlchemy==1.3.0
```
Comment (boecklic) [inline requirements.txt]: does it make sense to update directly to the latest minor release, 1.3.4?

Comment (procrastinatio) [reply requirements.txt]: yes

File: chsdi/models/__init__.py (hunk 3 above)
Comment (procrastinatio) [reply chsdi/models/__init__.py]: OK, done.

File: tests/integration/test_features_service.py (hunk 2 above)
Comment (procrastinatio) [reply tests/integration/test_features_service.py]: OK, added two tests (for EsriJson and GeoJson)

File: chsdi/lib/helpers.py (hunk 1 above)
Comment (procrastinatio) [reply chsdi/lib/helpers.py]: OK, makes sense

**Pull request merged**